src/quote/indexQuoteProvider.ts

```diff
@@ -122,6 +131,48 @@ export class FlashMintQuoteProvider
           tx,
         }
       }
+      case FlashMintContractType.leveragedExtended: {
+        if (!zeroExApiV1) {
+          throw new Error('Contract type requires ZeroExApiV1 to be defined')
+        }
+        const leverageExtendedQuoteProvider =
+          new LeveragedExtendedQuoteProvider(provider, zeroExApiV1)
+        const leveragedExtendedQuote =
+          await leverageExtendedQuoteProvider.getQuote(request)
+        if (!leveragedExtendedQuote) return null
+        const builder = new LeveragedExtendedTransactionBuilder(provider)
+        const txRequest: FlashMintLeveragedExtendedBuildRequest = {
+          isMinting,
+          inputToken: inputToken.address,
+          inputTokenSymbol: inputToken.symbol,
+          outputToken: outputToken.address,
+          outputTokenSymbol: outputToken.symbol,
+          inputTokenAmount: BigNumber.from(0), // TODO: leveragedExtendedQuote.inputTokenAmount,
+          outputTokenAmount: BigNumber.from(0), // TODO: leveragedExtendedQuote.outputTokenAmount,
```

Comment:
TODO: return inputTokenAmount + outputTokenAmount from quote as convenience properties for input here. At the moment they are still called `indexTokenAmount` and `inputOutputTokenAmount` in the return object of the  LeveragedExtendedQuoteProvider.